public/js/endpoints.js

```diff
@@ -16,6 +15,9 @@ $(document).ready(function() {
             case 'uninstall_chipsec':
                 api_url = `/api/chipsec/uninstall/${endpoint_host_id}`;
                 break;
+            case 'delete_host':
+                api_url = `/api/agents/rmNode/${endpoint_host_id}`;
```

Comment:
I think it's better to follow urls without capital letters. (it's not common) `/api/agents/delete` is more RESTful
I noticed that we have got a few routes with capitals, so consider changing it some time in the future.

Reply:
next time :)